contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_pair_values_to_have_diff.py

```diff
@@ -73,9 +69,12 @@ def _validate(
     ) -> Union[ExpectationValidationResult, dict]:
         diff: Union[float, int] = configuration["kwargs"].get("diff")
         mostly: str = configuration["kwargs"].get("mostly")
-        query_result: Union[sqlalchemy_engine_Row, pyspark_sql_Row] = metrics.get(
-            "query.column_pair"
-        )
+        query_result = metrics.get("query.column_pair")
```

Comment:
question : why don't we do the same `convert_to_json_serializable` step here?

Reply:
@Shinnnyshinshin It is not always needed, so reducing the need for this relatively slow serialization step whenever possible!  Thanks. :)